plugins/design-system/styles/styles.less

```diff
@@ -0,0 +1,6 @@
+@import '../../../node_modules/codemirror/lib/codemirror.css';
+@import 'components/component-example/styles.less';
```

Comment:
@bstavroulakis The PR you are looking at depends on a few other PRs, so it won't build until those other PRs are merged. The working branch is `tommy/component-example`. It is rather large, so I broke it up into multiple PRs.